packages/wonder-blocks-forms/generated-snapshot.test.js

```diff
@@ -0,0 +1,87 @@
+// This file is auto-generated by gen-snapshot-tests.js
+// Do not edit this file.  To make changes to these snapshot tests
+// edit packages/wonder-blocks-forms/docs.md and run `npm run gen-snapshot-tests`.
+import React from "react";
+import renderer from "react-test-renderer";
+import Dropdown from "./components/dropdown.js";
+
+describe("wonder-blocks-forms", () => {
+    it("example 1", () => {
+        const example = (
+            <Dropdown
+                items={[
+                    {label: "item 1", value: 1},
+                    {label: "item 2", value: 2},
+                    {label: "item 3", value: 3},
+                ]}
+            />
+        );
+        const tree = renderer.create(example).toJSON();
+        expect(tree).toMatchSnapshot();
+    });
+    it("example 2", () => {
+        const example = (
+            <Dropdown
+                show={true}
+                items={[
+                    {label: "item 1", value: 1},
+                    {label: "item 2", value: 2},
+                    {label: "item 3", value: 3},
+                ]}
+            />
+        );
+        const tree = renderer.create(example).toJSON();
+        expect(tree).toMatchSnapshot();
+    });
+    it("example 3", () => {
+        class Parent extends React.Component {
```

Comment:
I had to remove `// @flow` from this file b/c we can add prop flow typing within docs.md and then this fails to pass `flow`'s type-checking... sadness.